flow-state/store/store.go

```diff
@@ -19,6 +19,7 @@ const (
 )
 
 type Store interface {
```

Comment:
@milanbhagwat I think should consider creating different interface for DB.
e.g. type DBStore interface {
  Store
  GetMaxOpenConn() int
 GetDBPingStatus() bool
}